### DISCUSSION TITLE
New Roles in OMI and a Call For Volunteers

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/omigroup#165] [General]
Hey everyone 👋 As discussed in our meeting today, we think there could be more volunteer roles for people looking to get more involved in OMI. We have an election coming up on February 2nd and we'd like to get feedback from the community on how we'd like to proceed as a group. Based on the [task list Jesse and I compiled](https://hackmd.io/bDrL7EC6THSEU_RU7SsNXw), here are some positions that I think might fit us better than 3 general chairs. I think one person should be able to hold multiple of these roles. Currently the 3 chairs handle a couple of these responsibilities at a time, but breaking it out into more roles allows us to better delegate responsibilities.

### Steering Committee (3-5?):
  - Handles inbound partnership opportunities and reaches out to potential partners to further OMI's mission
  - Made up of a group of people who can speak on behalf of OMI on various topics
  - Communicates new partnership opportunities with the group and represents them when talking to prospective partners
  - Responsible for managing OMI's budget and assets based on the decisions set out by the group
  - Responsible for ensuring GitHub issues/discussions dealing with partnerships, budget, or organization responsibilities are brought to the group's attention and are resolved.

### General Meeting Moderators (2-3):
 - Helps create the agenda with the other moderators
 - Runs through the agenda
 - Moderates the call:
   - Manages speaker queue when necessary
   - Ensures people are given their time to speak
   - Invites others to join in on the conversation
   - Mutes users if they are having microphone issues
   - Enforces our Code of Conduct during the call
   - Runs votes during the call
 - Responsible for ensuring the meeting is recorded and notes are being taken. They don't have to be the ones taking notes/recording but they do need to make sure it's happening and they are distributed publicly.
 - Regularly runs retrospective meetings to check in on the health of our community
 - Responsible for ensuring GitHub issues/discussions dealing with new proposals for events, working groups, or other work items are brought to the group's attention and resolved.
 
### Working Group Chairs (1-2 per group):
  - Same as the general meeting but applied to individual working groups
  - Also responsible for adhering to the W3C process and ensuring that proper IP protection policies are followed
  - Listed under the W3C metaverse-interop group as chairs
  - Responsible for moderating and organizing the working group's GitHub

### Social Media Manager:
  - Our public voice for OMI on social media
  - Sends out updates on meetings, events, and projects happening within our communities
  - Creates and maintains content that is in line with our community's values
  
### Event Manager: 
  - Helps schedule and run regularly occurring events within our community
  - Examples include Demo Days, Twitter Spaces, Hackathons, Field Trips, etc.
  
### Discord Moderators (as many as we want):
  - Welcomes new users and helps them find their way in the community
  - Enforces our Code of Conduct throughout the server

## Summary

In summary, I think we still need a leadership body. I think that we should continue to divide those responsibilities and power. I think that a steering committee will be able to handle these outward facing interactions with other organizations that require us to have one or two people speaking on behalf of the group at a time. This comes up when responding to press, speaking opportunities, and in negotiating partnerships.

I think we should split up the other tasks the chairs and other members of our community have taken on into explicit roles which I've outlined above. I'm not sure that these encompass everything we need yet, but it's what I can come up with now and I'd love to hear from all of you on how you might structure this differently.

**If you are interested in filling one of these roles, please nominate yourself here! We'd love the help!**

Please, if you have time to chime in on this subject, do so before our next meeting on January 19th so we can discuss this then.

Thanks everyone!

**Comment by robertlong** [comment]
Oh, also I think we should separate the Steering Committee roles from W3C chairs. This will better allow us to take on leaders who do not feel comfortable with the W3C chairing process which requires a legal identity versus allowing for anonymous participants. We have a few people within this community who value their privacy and yet are trusted members of our community who should not have to be excluded from leadership opportunities.

I think W3C chairs should be reserved for chairs of the respective sub-groups where specifications are being authored as they align better with what W3C really does.

OMI as an org can then operate independently from W3C and only use W3C as an umbrella for specifications, etc.

**Comment by nathangrotticelli** [comment]
+1 to Robert's reply on Steering Committee sub-group affiliation
+ non-identity requirements.

**Comment by antpb** [comment]
Echoing the above in Roberts proposal, no one person should feel bound to only one role. Everyone should also feel able to step down and back in as they are comfortable and available.

With the extensions group as an example I could see chairs volunteering themselves as maintainers of certain components to give updates in meetings and verify process and stages of an extension lifecycle. Audio Emitter has been naturally taking shape. That's one component for instance I personally would be comfortable maintaining and shepherding implementations going forward. I think as more glTF extensions are added the group will need much more help organizing the different stages of proposals and implementations. All of those processes can come with time as we learn more about the role in practice, but that is one area I think would need 2 or more folks as chair representation to start. These reps should make it a goal of the term to document the tips/responsibilities for the next election. (something I'd feel eager and comfortable taking on for gltf-extensions if it makes sense and is agreed by the group 😄  )

I think one way to avoid perceived power and a need for term limits would be to give a cool title to people that have served in a role and stepped down. They helped make the metaverse more open and they should be able to celebrate that without having to hold a position.

**Reply by mrmetaverse** [reply]
>I think one way to avoid perceived power and a need for term limits would be to give a cool title to people that have served in a role and stepped down. They helped make the metaverse more open and they should be able to celebrate that without having to hold a position.

:point_up: love that idea :)

**Reply by jsejms** [reply]
> > I think one way to avoid perceived power and a need for term limits would be to give a cool title to people that have served in a role and stepped down. They helped make the metaverse more open and they should be able to celebrate that without having to hold a position.
> 
> ☝️ love that idea :)

+1 on that, great idea

**Comment by RedEagleP1** [comment]
If I may comment, the individual responsibilities seem to great for volunteers like us who only have limited time due to work. For example, I would be willing to run some meetings but I can't do them all since sometimes work interferes time-wise. My suggestion would be to specify limited roles for those who love the vision but can't put in the same amount of time the founders do.

**Comment by RedEagleP1** [comment]
"General Meeting Moderators" I'd be willing to volunteer to lead a meeting 1/2 times per month if some previsions can be made in case of emergency.

**Comment by Elirudite** [comment]
I'd like to propose that instead of electing new chairs, we instead take the list of chair responsibilities and break it up into discrete tasks that anybody from the community can assign themselves to do. This would solve the problem of unsustainable workload falling on the leaders, make it easier for new/old members to see exactly what they can do to get involved and contribute, as well as create more clear/open space for accountability + gratitude for the work people are doing here.
This can be a 3-month 'sprint' for us to test how well (or not) this method of organizing works.
